### PR TITLE
Add data for BAVcombo24

### PR DESCRIPTION
**Pull request by smaugustine**
The catalogue contains very little details, including having nothing to say about the collation. I could add the collation based on the photographs, if that's worthwhile to do. There were also some mistakes in the incipit (which I have corrected, namely, በኃይላተ>በኆኅያተ and ሃደሉ>ፊደሉ).

### CODE REVIEW

**Comment by PietroLiuzzo** [comment]
since images are available, if you want I can run this in transkribus to get the full transcription (to be fixed). let me know

**Comment by smaugustine** [comment]
I have it partially included in the TEI file of my preliminary crit. edition -- if I were to finish adding it to the `<app>`s then that could be used for the transcription, but I'm hesitant since I'm hoping to eventually publish the edition.

**Comment by smaugustine** [comment]
It's not an easy one to read either, I would expect a lot of errors from an automated transcription.

**Comment by PietroLiuzzo** [comment]
too much missing info. name me a page and I can do a quick test. Transkribus has indeed given surprisingly good results.

**Comment by smaugustine** [comment]
Page 40 (img 0044)?

**Comment by eu-genia** [comment]
When catalogues don't give collation (or any other information) we don't have to do it at this stage, unless this is a manuscript you need in your work and would like to analyse in detail.

File: VaticanBAV/comb/BAVcombo24.xml [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+                    </msIdentifier>
+
+                    <msContents>
+                      <summary/>
+
+                      <msItem xml:id="ms_i1" class="content">
+                        <locus from="3r" to="3v" facs="0009">5-6</locus>
+                        <title type="complete" ref="LIT5888AsmatPrayer"/>
+                        <textLang mainLang="gez"/>
+                        <incipit xml:lang="gez">በስመ፡ <gap reason="omitted" extent="unknown"/> ህቡዕ፣ ስሙ፡ ለእግዚአብሔር፡ አስማተ፡ ኃይላት፡ ኢያኤል፡ አፍናኤል፡ አስናኤል፡ አትናኤል፡ ስንኰራኵኤል፡ ፍዳኤል፡ አፍናኤል፡ ኦእግዚእየ፡ ኢየሱስ፡ ክርስቶስ፡ እቀቦ፡ ለግብርከ፡ <del rend="erasure"/></incipit>
```
Comment (eu-genia) [inline VaticanBAV/comb/BAVcombo24.xml:38]: you need `<gap reason="ellipsis">` for abbreviations on the part of the editor (I assume Raineri simply skips the trinitarian formula)
https://betamasaheft.eu/Guidelines/?q=omitted&id=text-encoding

File: VaticanBAV/comb/BAVcombo24.xml [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+                      </msItem>
+
+                      <msItem xml:id="ms_i2" class="content">
+                        <locus from="4r" to="26r" facs="0011">7-51</locus>
+                        <title type="complete" ref="LIT3034RepCh314"/>
+                        <textLang mainLang="gez"/>
+                        <incipit xml:lang="gez">በስመ፡ <gap reason="omitted" extent="unknown"/> ጸሎት፡ በእንተ፡ ልሳነ፡ ሰብእ፡ ዘመድ፡ ወባዕድ፡ መቀኛ፡ ወተንኰለኛ፡ ቡዳ፡ ወእደ፡ ሰብእ። ናሁ፡ ተማኅፀንኩ፡ በኆኅያተ፡ ስምከ፡ ካፍ። ወበቀዳማይ፡ የውጣ፡ ዘጥንተ፡ ፊደሉ፡ አሌፍ፡ ከመ፡ ታድኅነኒ፡ ክርስቶስ፡ <gap reason="omitted" extent="unknown"/></incipit>
```
Comment (eu-genia) [inline VaticanBAV/comb/BAVcombo24.xml:45]: see above

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Comment by PietroLiuzzo** [comment]
> Page 40 (img 0044)?

I hope I got the right one... current model (claiming 92% CER) returns this

```xml
<pb n='1'/>
<lb  n='1'/>ሳናሩ፡ ወአመልዕነ፡ ዕበአ 
<lb n='2'/>ቢራን፡ ተተማኅፀዓኩ፡ ክርስ
<lb n='3'/>ብወስቀልከ፡ መድኅንን፡ እን
<lb n='4'/>መ፡ አመልሳን፡ ወጽአ፡ ኵለ
<lb n='5'/>ሕዘን። ሶበሰ፡ ኢተፈጥሮ፡ ለ
<lb n='6'/>ተናግር፡ ልሳን፨ አስ፡ ቶ
<lb n='7'/>ት፡ አዳመ፡ በመንተ፡ እም 
<lb n='8'/>ስህለ፡ ሰይጣዓ። ይ
```

File: VaticanBAV/comb/BAVcombo24.xml [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+                    </msContents>
+                    <physDesc>
+                      <objectDesc form="Codex">
+                        <supportDesc>
+                          <support>
+                            <material key="parchment"/>
+                          </support>
+                          <extent>
+                            <measure unit="leaf">30</measure>
+                            <dimensions type="outer" unit="cm">
```
[the remark this answers is not in the document]
Comment (smaugustine) [reply VaticanBAV/comb/BAVcombo24.xml:58]: The guidelines state that if the catalogue uses another unit they should be maintained.

File: VaticanBAV/comb/BAVcombo24.xml (hunk 1 above)
Comment (smaugustine) [reply VaticanBAV/comb/BAVcombo24.xml:38]: Okay. Other examples on BetMas had "omitted", so I have been using that. I'll change the other BAV records where I had "omitted", then.

**Comment by smaugustine** [comment]
This page is a bit easier (I was thinking of four pages earlier, with many stains and smudges).

`
<lb n='1'/>ሳናት፡ ወእምልሳነ፡ ሰብእ፡ ነ
<lb n='2'/>ኪራን፨ ተማኅፀንኩ፡ ክርስ
<lb n='3'/>በመስቀልከ፡ መድኅን፨ እስ
<lb n='4'/>መ፡ እምልሳን፡ መጽአ፡ ኵሉ
<lb n='5'/>ሐዘን፨ ሶበሰ፡ ኢተፈጥሮ፡ ለ
<lb n='6'/>ተናግሮ፡ ልሳን፨ አስሕቶ
<lb n='7'/>ተ፡ አዳም፡ በምንት፡ እም
<lb n='8'/>ክህለ፡ ሰይጣን፨ እ
`

**Comment by PietroLiuzzo** [comment]
```xml
<pb n='2'/>
				<lb n='1'/>አርለ፡ ቅለቀ፡ ዓአመ፡ በልባል፡  
				<lb  n='2'/>ብነ፡ እስመ፡ አቀብቱ፡ ዝኅር
				<lb  n='3'/>ረናበ፡ ከከመ፡ አብድንቱ፡ ዘ 
				<lb   n='4'/>ዩ፡ አድጥኪ፡ ክርስ፡ ሰብአ፡ል 
				<lb  n='5'/>ሰላምዐዐ፡ እመሠልጣሩ፡ ለክ 
				<lb  n='6'/>በሰዊዎ፡ ዘአቅተአ፨ ወአዓ፡  
				<lb  n='7'/>በእሴ፡ ሰላመደ፡ ይሁዱ፡ አከ፡
				<lb  n='8'/>ቅኑቱ፡ በዕርገቶስ፡ ወለጽ 
```

**Review by eu-genia — APPROVED** [review]

**Pull request merged**